src/lib/radio/radio.spec.ts

```diff
@@ -316,7 +316,7 @@ describe('MatRadio', () => {
 
     it('should not fire a change event from the group when a radio checked state changes', () => {
```

Comment:
Can you update the test names from `change`  to `selectionChange` as well?

Reply:
Updated